neurox/data/filter.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+import codecs
+import h5py
+import json
+import re
+import collections
+import numpy as np
+
+"""Create a binary word-level data set using a set of positive examples, or a regular expression or a function
```

Comment:
Move this to top for it to become "module" documentation. See top of https://github.com/fdalvi/NeuroX/blob/master/neurox/data/writer.py